contrib/qa/test-binary-availability.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash 
+
+# Local builds and Travis CI builds of the BitcoinCore.org website do
```

Comment:
Isn't that harder to read?  DNS is case insensitive, so both resolve to the same location, and it's always been my habit to capitalize domain names as proper nouns in paragraph text.  I'm happy to paint this bikeshed any color you want, I'm just a little curious why it has to be hot pink. :-)

Reply:
no, any colour is fine :)

my general preference is for all lowercase since that seems like the normalized version to me, but if your convention is to uppercase proper nouns that's also fine.